src/App.js

```diff
@@ -17,11 +12,9 @@ class App extends Component {
         <header className="App__header">
           <h1 className="App__title">Coding Coach</h1>
           <h4 className="App__tagline">Connecting developers with mentors worldwide.</h4>
-          <button className="button button__white--fill" onClick={this.onGetStarted}>
-            <a className="button__link">
-              Get Started
-            </a>
-          </button>
+          <a className="button button__white--fill button__link">
```

Comment:
Hi, thanks for your contribution. But, it seems like this `<a>` element is missing the `href` attribute now which previously pointed to the google docs. Could you please verify that once?

Regards!

Reply:
I forgot to add back, fixed.